app/views/administration/externalServices.html

```diff
@@ -27,9 +27,7 @@ <h4 class="list-group-item-heading"><i class="fa fa-large fa fa-envelope"></i>&n
                         </div>
                     </div>
                 </div>
-            </div>
```

Comment:
did we break this.. i don't see card being closed </card>

Reply:
this card has been closed at the bottom --line no. 62 for proper arrangements.